cmd/config/config.go

```diff
@@ -275,6 +275,11 @@ func ReadMountSpec(r *v1.RunConfig, flags *pflag.FlagSet) (*v1.MountSpec, error)
 		return mount, err
 	}
 
+	if err := applyMountEnvVars(r, mount); err != nil {
```

Comment:
Not 1005 sure, but shouldn't this call happen before the `applyKernelCmdline`? this way kernel parameters have priority. I think it was done this way before.

Reply:
Good point! It's hard to tell in the old code but I will try it on a fresh system and see which takes precedence!

Reply:
Hmm, it looks like the values in `/run/cos/cos-layout.env` have been picked over kernel-cmdline previously. Right now it's only actually used for the ephemeral overlay settings so I'm inclined to use the kernel-cmdline if it exists.. WDYT?

Reply:
Yes I agree :+1: